install_config/cluster_metrics.adoc

```diff
@@ -598,7 +598,7 @@ clusterInfo:
 ====
 
 Once you have updated and saved the webconsole-config configmap file, you must
-restart your {product-title} instance.
+restart your webconsole pods.
```

Comment:
s/webconsole/web console

It looks like this is two word in normal text. If "webconsole" is the name of the pod, you could s/webconsole/\*webconsole\* instead.

Reply:
@kalexand-rh Thank you for your review as always. "webconsole" points the name of the pod, but I think "web console" is better term to point the pod usually. I've modified `s/webconsole/web console/`.